tests/tests_default_reboot.yml

```diff
@@ -12,10 +14,97 @@
 
     - name: Check generated files for ansible_managed, fingerprint
       include_tasks: tasks/check_header.yml
-      loop:
-        - /etc/kdump.conf
-        - /etc/sysconfig/kdump
-      loop_control:
-        loop_var: __file
       vars:
         __fingerprint: "system_role:kdump"
+        __file: /etc/kdump.conf
+
+    - name: See if el9 or later
+      set_fact:
+        is_el9: "{{
+          ansible_facts['distribution'] in ['RedHat', 'CentOS', 'Fedora']
```

Comment:
Fedora is redundant here? You can set is_el8 too, the check for RHEL 8 is used twice below.